src/orbits/kepler.jl

```diff
@@ -1,110 +1,215 @@
+using AstroLib: kepler_solver, trueanom
+using KeywordDispatch
+using PhysicalConstants
+using Unitful
+using UnitfulAstro
 
-using AstroLib: kepler_solver
+const G = PhysicalConstants.CODATA2018.G
+const G_val = 6.67430e-8 # CGS
```

Comment:
You can get the value from `G`:
```julia
julia> ustrip(u"cm^3/g/s^2", CODATA2018.G)
6.674299999999999e-8
```
```suggestion
const G_val = ustrip(u"cm^3/g/s^2", G) # CGS
```